app-config.yaml

```diff
@@ -12,7 +12,7 @@ backend:
   # auth:
   #   keys:
   #     - secret: ${BACKEND_SECRET}
-  baseUrl: /
+  baseUrl: http://localhost:7007
```

Comment:
This doesn't seem right.

Reply:
I really don't remember... I think we were overriding this value

Reply:
I'm convinced it's right.  :)

Reply:
I'm also convinced it's weird.  lol